### PR TITLE
As per meeting notes of Jan 17, reword the 25% requirement.

### PR DESCRIPTION
**Pull request by jmlait**
As per meeting notes of Jan 17, reword the 25% requirement.

### CODE REVIEW

**Comment by kmuseth** [comment]
May I suggest a minor rewording:

"an active contributor to the project in general and the code base in particular"

**Comment by jmertic** [comment]
I think this is reasonable

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The committer role enables the participant to commit code directly to the reposi
 * Monitor Slack (delayed response is perfectly acceptable).
 * Triage GitHub issues and perform pull request reviews for other committers and the community.
 * Make sure that ongoing PRs are moving forward at the right pace or close them.
-* In general continue to be willing to spend at least 25% of one's time working on the project (~1.25 business days per week).
+* In general continue to be an active contributor to the project in general and the code base in particular.
```
Comment (danrbailey) [inline CONTRIBUTING.md]: In general ... in general ?

Comment (kmuseth) [reply CONTRIBUTING.md]: good catch - let's change it to:

"* Remain an active contributor to the project in general and the code base in particular."

Comment (jmlait) [reply CONTRIBUTING.md]: But.... Then we lose the wonderful passive voice?  :>

I've made that change.

**Comment by kmuseth** [comment]
just confirming that  I'm good with Jeff's latest edits

**Pull request merged**